### PR TITLE
ci: various fixes

### PR DESCRIPTION
**Pull request by alexeykuzmin**
##### Description of Change

 - ci: don't send Slack notifications for the "linux-ia32-testing-tests" job
 - ci: fix symbols collection job on VSTS
 - ci: run tests on Windows for both "testing" and "release" builds

**nightly-release-osx-x64**
https://github.visualstudio.com/electron/_build/results?buildId=13940&view=logs

**electron-nightly-release-test-x64**
https://windows-ci.electronjs.org/project/AppVeyor/electron-hernh/build/1.0.106

##### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)

##### Release Notes

Notes: no notes

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -51,16 +51,20 @@ build_script:
         appveyor PushArtifact out/Default/symbols.zip
       }
 test_script:
-  - if "%GN_CONFIG%"=="testing" ( echo Verifying non proprietary ffmpeg & python electron\script\verify-ffmpeg.py --build-dir out\Default --source-root %cd% --ffmpeg-path out\ffmpeg )
   - ps: >-
-      if ($env:GN_CONFIG -eq 'testing') {
+      if ($env:GN_CONFIG -eq 'testing' -Or $env:GN_CONFIG -eq 'release') {
```
Comment (MarshallOfSound) [inline appveyor.yml]: This will run the tests for the publish / actual releasing builds which we don't want.

**Comment by alexeykuzmin** [comment]
> Need to be sure we don't end up running tests during publish builds

@MarshallOfSound Do we run tests on the same commit before publishing?

**Comment by MarshallOfSound** [comment]
> Do we run tests on the same commit before publishing?

Yes, tests are run on all commits in PR's and release branches

**Comment by alexeykuzmin** [comment]
Anyway. I preserved the current behaviour.

**Comment by alexeykuzmin** [comment]
> Yes, tests are run on all commits in PR's and release branches

It is not enough. CI environment might get changed.
We should test exactly the same binary we are shipping.

**Comment by alexeykuzmin** [comment]
The tests ran for **appveyor: electron-nightly-release-test-x64**, the "Collect symbols" step for **nightly-release-osx-x64** succeeded, merging with those builds failed anyway.

**Pull request merged**